### DISCUSSION TITLE
Branding and auth provider limitations

### DISCUSSION THREAD

**Discussion by DeadGhost1407** [lowcoder-org/lowcoder#207] [Q&A]
Hi Lowcoder team, I am glad you are continuing the openblocks work and wish you best of luck. 

Are you planning on keeping the limitations on branding and auth providers under an enterprise/premium license?

**Comment by FalkWolsky** [comment]
Yes / No... But way more "No" then yes.
What do we mean: It is our interest that everyone can make great apps and a good Business with Lowcoder. We will do everything for that. In favour to branding, we reduced the Lowcoder Branding so much, that in the Community Edition only a small "powered by lowcoder" is actually left. Everything else is neutral.

Also, by introduction of the generic OAuth Provider with Lowcoder v2.4.0 we opened Lowcoder to be used with pretty much all OAuth Providers thinkable. Both steps are realized in the free Community Edition.

Next to that we will offer more branding options as own Logo Placements, Colors and other Branding Elements in the Enterprise Version. This is typically needed for large Companies. Also the support for SSO / SAML and LDAP is more relevant for big companies - so we also see this in the EE Version.
